### ISSUE TITLE
Improve Widevine installation for Windows and fix FAQ

### ISSUE THREAD

**Issue by vairag22** [Eloston/ungoogled-chromium#1439]
**TL;DR -> Just goto woolyss' [site](https://chromium.woolyss.com) and download the version of Chromium you want. He also has a better wiki.**

The [official guide for installing Widevine on Chromium Windows](https://ungoogled-software.github.io/ungoogled-chromium-wiki/faq#windows) has some issues. Following are the problems in each step:

**Problem in Step 1**: _current.txt_ which is supposed to have info on latest version of Widevine is severely outdated - Widevine version in _current.txt_ is 1.4.9.1088, while in Firefox it is 4.10.1582.2 - and by following Step 2 user will download an outdated version.

**Problem in Step 3**: Instructions aren't clear enough imo. Just say create _WidevineCdm_ folder inside your Chromium directory.

**Problem in Step 4**: Fix typo in first line: _**_plaform_specific**_ -> _**_platform_specific**_.  One could mess up the folder name if he copy-pastes without noticing the error, which I did.

**Problem in Step 5**: It isn't clear what user should paste in WidevineCdm and win_x64 directory. Should I place _manifest.json_ or _LICENSE.txt_, or both in WidevineCdm folder and should I place _widevinecdm.dll_ or _widevinecdm.sig_ (typo alert: it should be_widevinecdm.dll.sig_), or both in win_x64 folder. **Solution: Paste _manifest.json_ and _LICENSE.txt_ in _WidevineCdm_ folder, and _widevinecdm.dll_ and _widevinecdm.dll.sig_ in _win_x64_ folder.**

**Comment by vairag22** [comment]
I provided a solution as a feature request #1440

**Comment by vairag22** [comment]
merged #1440 with #1439 (this issue)

**Comment by r4sas** [comment]
PR changes here: https://github.com/ungoogled-software/ungoogled-chromium-wiki/blob/master/faq.md#windows

**Comment by woolyss** [comment]
@vairag22:
There is a tutorial at https://chromium.woolyss.com/#widevine if needed.
All the links are official and the Widevine version is auto-updated.

@Eloston: 
If you want to update your wiki....copy-paste or for inspiration  ;)

**Comment by rany2** [comment]
You could do something similar to https://github.com/flathub/com.github.Eloston.UngoogledChromium/blob/master/widevine-install.sh .

I just download https://dl.google.com/widevine-cdm/current.txt and get the last line for the version
